src/test/java/com/stripe/model/EventDataDeserializerTest.java

```diff
@@ -0,0 +1,24 @@
+package com.stripe.model;
+
+import static org.junit.Assert.assertNotNull;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.net.APIResource;
+
+import org.junit.Test;
+
+public class EventDataDeserializerTest extends BaseStripeTest {
+
+  @Test
+  public void testApplicationEventData() throws Exception {
```

Comment:
Can you rename this to something like `testEventAccountApplicationDeauthorized` to make it clear that the test is only for a specific event type? (In the future, we could add more tests for different event types.)